omnibus/files/private-chef-cookbooks/private-chef/libraries/helper.rb

```diff
@@ -112,18 +112,11 @@ def elastic_search_major_version
   end
 
   def es_index_definition
-    if node['private_chef']['elasticsearch']['first_internal_install'] == true
-      es_6_index
-    else
-      # For external elasticsearch or chef-backend elasticsearch will be running
-      # before we try to create the index
-      es_version = elastic_search_major_version
-      if es_version == 6
+      if elastic_search_major_version == 6
         es_6_index
       elsif [2, 5].include?(es_version)
```

Comment:
`es_version` will be undefined here. I think this should be `elatic_search_major_version`

Reply:
I am sure I have pushed this change twice - not sure whay it did not reflect. Will check.